src/main/java/model/EventModel.java

```diff
@@ -0,0 +1,47 @@
+package model;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import dao.DaoFactory;
+
+public class EventModel {
+	
+	private DaoFactory daoFactory;
+	
+	public EventModel() {
+		this.daoFactory = new DaoFactory();
+	}
+	
+	public ArrayList<HashMap<String, Object>> getSportsCitiesList(String object) {
+		ResultSet list = null;
+		if(object == "city") {
+			list = daoFactory.getEventDao().getCities();		
+		}
+		if(object == "sport") {
+			list = daoFactory.getEventDao().getSports();
+		}
+		ArrayList<HashMap<String, Object>> result = getResultList(list, object);
+		return result;
```

Comment:
1. I would recommend using equals for string comparison as == can sometimes create seemingly odd bugs, == compares a reference while equals compares values (or whatever you want if you override it) e.g. if you have 2 classes with a string str="test" and you compare these strings like objectA == objectB it will return false as the value comes from different object.
2. this method is unnecessary, you can simply have two methods one for sports and one for cities and it will remove the necessity for string comparison
3. Why even initialize result list if you do nothing with it return it immediately in the next line, you can simply "return getResultList"  
4. Why even pass "object" string further, it should not be used elsewhere